README.md

```diff
@@ -83,11 +79,11 @@ Minimal dependencies are installed without extra options. You can install extra
 pip install "pyautogen[blendsearch]"
 ``` -->
 
-Find more options in [Installation](https://microsoft.github.io/autogen/docs/Installation).
+Find more options in [Installation](https://microsoft.github.io/autogen/docs/Installation#option-2-install-autogen-locally-using-virtual-environment).
 
 <!-- Each of the [`notebook examples`](https://github.com/microsoft/autogen/tree/main/notebook) may require a specific option to be installed. -->
 
-For [code execution](https://microsoft.github.io/autogen/docs/FAQ/#code-execution), we strongly recommend installing the Python docker package and using docker.
+Even if you are installing AutoGen locally out of docker, for better user experience and seamless code execution, we recommend performing [code execution](https://microsoft.github.io/autogen/docs/FAQ/#code-execution) in docker. To do so, you will need to [install Python docker package](https://microsoft.github.io/autogen/docs/Installation#docker) and set `use_docker` to `True` in `code_execution_config`.
```

Comment:
1. I'm not sure docker gives "better user experience and seamless code execution".
2. Remove "set `use_docker` to `True` in `code_execution_config`".
3. It's not enough to install the python docker package.